app/controllers/api/v2/courses_controller.rb

```diff
@@ -195,7 +225,12 @@ def enrichment_params
                   :study_mode,
                   :is_send,
                   :accrediting_provider_code,
-                  :funding_type)
+                  :funding_type,
+                  :name,
+                  :course_code,
+                  :subjects_ids,
+                  :subjects_types,
+                  :level)
```

Comment:
What if we had a list of these attribute names in a variable `enrichment_update_attributes`, and used that here and below in `course_params`. Likewise for `course_update_attributes`, and we wouldn't get out of sync like this.

Reply:
What would we do with them? Not all of these parameters are relevant to all functions.